content/blog/2020-06-28-shtab-completion.md

```diff
@@ -0,0 +1,186 @@
+---
+title: '(Tab) Complete Any Python Application in 1 Minute or Less'
+date: 2020-06-28
+description: |
+  We've made a painless tab-completion script generator for Python applications!
+  Find out how to take advantage of it in this blog post.
+
+descriptionLong: |
+  We've made a painless tab-completion script generator for Python applications!
+  It's called `shtab` and it currently works with `argparse`, `docopt`, and
+  `argopt` to produce `bash` and `zsh` completion scripts.
+  This tool was originally created to help `dvc`, but we realised it could be made
+  more generic and valuable to the world's entire ecosystem of Python CLI
+  applications. Find out how to take advantage of it in this blog post.
+author: casper_dcl
+---
+
+> Other potential titles:
+>
+> - Zero Effort Tab Completion for Python Applications
+> - Finally: 1-Click Tab Completion for [`argparse`], [`docopt`] and [`argopt`]
+> - A [`shtab`] in the dark at completing your ~~sentences~~ commands
+
+Command line tools are powerful. Things like [`make`] have manual pages
```

Comment:
make a link to make man?

Reply:
https://man7.org/linux/man-pages/man1/make.1.html or https://www.gnu.org/software/make/manual/make.html?

Reply:
I'm fine with either :)

Reply:
added

Reply:
No link yet 🙂

Reply:
Ah never mind I see the links are indexed in the bottom!